### PR TITLE
Problems solved on 9/22

### PR DESCRIPTION
**Pull request by HidemichiShimura**
These are problems on LeetCode which were solved on 9/22.

### CODE REVIEW

File: 1436.destination-city.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * @param {string[][]} paths
+ * @return {string}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/destination-city/
+*/
+
+var destCity = function(paths) {
+    const pathTable = {};
```
Comment (ryonryon) [inline 1436.destination-city.js:11]:
```suggestion
    const pathTable = new Map();
```

File: 1773.count-items-matching-a-rule.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            break;
+        case "color":
+            ruleKeyIndex = 1;
+            break;
+        case "name":
+            ruleKeyIndex = 2;
+            break;
+    }
+    
+    let count = 0;
```
Comment (ryonryon) [inline 1773.count-items-matching-a-rule.js:26]: the variable name is not self-descriptive.

File: 1832.check-if-the-sentence-is-pangram.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ * @param {string} sentence
+ * @return {boolean}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/check-if-the-sentence-is-pangram/
+*/
+
+ var checkIfPangram = function(sentence) {
+    const alphabetTable = {};
```
Comment (ryonryon) [inline 1832.check-if-the-sentence-is-pangram.js:11]:
```suggestion
    const alphabetMap = new Map();
```

File: 1832.check-if-the-sentence-is-pangram.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    
+    for (let i = 0; i < sentence.length; i++) {
+        if(alphabetTable[sentence[i]] === undefined) {
+            alphabetTable[sentence[i]] = 1;
+        }
+    }
+    
+    return Object.keys(alphabetTable).length === 26 ? true : false;
```
Comment (ryonryon) [inline 1832.check-if-the-sentence-is-pangram.js:19]:
```suggestion
    return Object.keys(alphabetTable).length === 26
```
this works the same.

File: 1436.destination-city.js (hunk 1 above)
Comment (HidemichiShimura) [reply 1436.destination-city.js]: I updated the initialization of pathTable as you described above.

File: 1773.count-items-matching-a-rule.js (hunk 2 above)
Comment (HidemichiShimura) [reply 1773.count-items-matching-a-rule.js]: I change the name of the variable to matchingItemCount.
Is this name self-descriptive enough?

File: 1832.check-if-the-sentence-is-pangram.js (hunk 3 above)
Comment (HidemichiShimura) [reply 1832.check-if-the-sentence-is-pangram.js]: I updated the name and the initialization of alphabetTable as you described above.

File: 1832.check-if-the-sentence-is-pangram.js (hunk 4 above)
Comment (HidemichiShimura) [reply 1832.check-if-the-sentence-is-pangram.js]: I removed the last part starting with ?.

**Comment by HidemichiShimura** [comment]
I appreciate your reviews.
Thank you so much.

**Pull request merged**